.github/workflows/retag_image.yml

```diff
@@ -47,17 +47,26 @@ jobs:
       - name: Retag image
         run: |
           set -x
-            
-          MANIFEST=$(aws ecr batch-get-image --repository-name ${{ secrets.AWS_RESOURCE_NAME_PREFIX }} --image-ids imageTag=${{ inputs.commit_id }} \
+
+          MANIFEST=$(aws ecr batch-get-image --repository-name $repo --image-ids imageTag=$commit_id \
```

Comment:
Where is $repo defined?  Doesn't it needs to be updated to use a secret.

Also doesn't $commit_id need to be updated?